### PR TITLE
some doc on a TeX bug

### PR DESCRIPTION
**Pull request by FrankMittelbach**
# Internal housekeeping

## Status of pull request

 Ready to merge

## Checklist of required changes before merge will be approved
- [na] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [x] Rollback provided (if necessary)?
- [??] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

Probably not the greated prose, but it is mainly there to remind 2-3 people in the world what this is all about and a few others to be careful not to remove something that looks useless

### CODE REVIEW

File: required/tools/array.dtx [hunk 1]
```diff
@@ -1386,33 +1400,112 @@ Bug reports can be opened (category \texttt{#1}) at\\%
...
-%    will fail; that can be fixed by adding a \cs{relax} after the \cs{@tempcnta},
-%    but that then leads to issues if you are collecting whole cells (tagging code
-%    or \\pkg{collcell}), where you can no longer alter the meaning of \cs{cr}
-%    as the master counter goes wrong.
+%    failed. That can be fixed by adding a \cs{relax} after the
+%    \cs{@tempcnta}, but that then leads to issues if you are
+%    collecting whole cells (tagging code or \pkg{collcell}), where
+%    you can no longer alter the meaning of \cs{cr} as the master
+%    counter goes wrong due to an obscure bug (or perhaps, say, an
+%    undocumented feature pf \TeX{}).  Eventually, we were able to pin
```
Comment (davidcarlisle) [inline required/tools/array.dtx:1431]: of

**Review by davidcarlisle — APPROVED** [review]

File: required/tools/array.dtx [hunk 2]
```diff
@@ -464,16 +478,16 @@
 %   "\newcolumntype{L}{>{$}l<{$}}" \\
 %   "\newcolumntype{R}{>{$}r<{$}}"
 % \end{quote}
-% Then we can use \texttt{C} to get centred LR-mode in an
-% \texttt{array}, or centred math-mode in a \texttt{tabular}.
+% Then we can use \texttt{C} to get centered LR-mode in an
+% \texttt{array}, or centered math-mode in a \texttt{tabular}.
 %
-% The example given above for `centred decimal points' could be
+% The example given above for `centre decimal points' could be
```
Comment (josephwright) [inline required/tools/array.dtx:484]:
```suggestion
% The example given above for `center decimal points' could be
```

**Review by josephwright — APPROVED** [review]
ON UK -> US missed, otherwise looks good

**Comment by josephwright** [comment]
> Probably not the greated prose, but it is mainly there to remind 2-3 people in the world what this is all about and a few others to be careful not to remove something that looks useless

At least four of us looked at this :)

File: required/tools/array.dtx [hunk 3]
```diff
@@ -1386,33 +1400,112 @@ Bug reports can be opened (category \texttt{#1}) at\\%
...
-%    but that then leads to issues if you are collecting whole cells (tagging code
-%    or \\pkg{collcell}), where you can no longer alter the meaning of \cs{cr}
-%    as the master counter goes wrong.
+%    failed. That can be fixed by adding a \cs{relax} after the
+%    \cs{@tempcnta}, but that then leads to issues if you are
+%    collecting whole cells (tagging code or \pkg{collcell}), where
+%    you can no longer alter the meaning of \cs{cr} as the master
+%    counter goes wrong due to an obscure bug (or perhaps, say, an
+%    undocumented feature of \TeX{}).  Eventually, we were able to pin
+%    down the the root cause and really understand why
```
Comment (Skillmon) [inline required/tools/array.dtx:1432]: "the the"

Comment (josephwright) [reply required/tools/array.dtx]: Fixed - there were two :)

**Pull request merged**